### PR TITLE
Feat/webp

### PR DESCRIPTION
**Pull request by imranhaidercogworks**
**Added support for WebP parameters and quality setting**

**What this PR does / why it's submitted / why we need it**:

---

**Checklist**:

- [X] My PR has a descriptive title (not a vague title like `Needed this`)
- [X] Title and all the commits follow [commitlint guidelines](https://github.com/conventional-changelog/commitlint#what-is-commitlint)
- [X] My PR targets the `develop` branch or appropriate `release` branch
- [ ] I've placed the link to the corresponding Trello card or issue which this PR fixes/address below ⤵️

---

Fixes:

### CODE REVIEW

File: Source/Cogworks.Umbraco.Essentials/Extensions/ImageCropperExtensions.cs [hunk 1]
```diff
@@ -34,20 +34,34 @@ public static string GetCropUrls(this IPublishedContent image, string cropAlias,
...
         }
 
-        private static string UpdateCropDimensions(string imageCrop, int? width, int? height)
+        private static string UpdateCropSettings(string imageCrop, int? width, int? height, bool enableWebP, int? quality = null)
         {
             var query = HttpUtility.ParseQueryString(imageCrop);
 
             if (width.HasValue)
             {
+                query.Remove(ImageCropConstants.Width);
```
Comment (lkoruba) [inline Source/Cogworks.Umbraco.Essentials/Extensions/ImageCropperExtensions.cs:43]: Is this necessairy to remove this?

**Review by lkoruba — APPROVED** [review]

**Comment by lkoruba** [comment]
@imranhaidercogworks should we install ImageProcessor package here to handle Webp or this is gonne be done on project level?

File: Source/Cogworks.Umbraco.Essentials/Extensions/ImageCropperExtensions.cs (hunk 1 above)
Comment (imranhaidercogworks) [reply Source/Cogworks.Umbraco.Essentials/Extensions/ImageCropperExtensions.cs:43]: I don't think it is necessary but it is probably safer

**Comment by imranhaidercogworks** [comment]
> @imranhaidercogworks should we install ImageProcessor package here to handle Webp or this is gonne be done on project level?

This just adds the WebP parameters to the querystring

The WebP settings are set to false by default so the user should only add the true setting if they have WebP capabilities

It also doesn't break the image if the querystrings are added without the ImageProcessor package

So I think we shouldn't add it in here as it is not required

**Review by aochmann — APPROVED** [review]

**Pull request merged**